### ISSUE TITLE
Cannot build from source

### ISSUE THREAD

**Issue by tymalo** [AAndyProgram/SCrawler#4]
Cloning the project in visual studio shows that its missing a file called InfoMod.vb and its also making many references to a different project namespace called PersonalUtilities. 

Due to the missing files you cannot build this, run it locally or create changes.

**Comment by AAndyProgram** [comment]
Delete the "PersonalUtilities" project and replace it with the latest version of the "PersonalUtilities" library (from the latest release).
Import PersonalUtilities.Functions for the whole project.

I don't know about InfoMod.vb (I don't actually remember this object), but all files should be present. I posted the complete code.
I will review this later (InfoMod.vb)

**Comment by AAndyProgram** [comment]
I found this module. It was not posted by mistake. I added one root folder "Info" to gitignore, which contains my notes. Unfortunately, the project has a folder with the same name. In older versions, this folder contained several modules with functions that were deprecated before the first public release and have been removed. Now there is only one function in this module to check for updates. In the next release, I'll move this function to the MainMod.vb file.
